src/scales/scale.ts

```diff
@@ -200,4 +200,15 @@ export class Scale<D, R> {
     this._autoDomainIfAutomaticMode();
     return this;
   }
+
+  /**
+   * Whether Scale is comparable intuitively. eg. Quantitative vs Category, Time vs Time.
```

Comment:
Sorry, I'm not clear what this means - intuitively I wouldn't imagine Scales to be comparable at all since they're functions. How does comparability relate to "find the right data point on hover"?